### DISCUSSION TITLE
Video url field when editing properties of an image

### DISCUSSION THREAD

**Discussion by adam-jones-net** [area17/twill#1802] [Q&A]
Hi all. I have an install that has been running a few years and the customer suddenly came to me asking about a Video URL field that they see when clicking the "Edit Info" link next to an image of a _medias_ field.

![image](https://user-images.githubusercontent.com/3810108/188966171-360174d2-fc9d-4428-8703-998d30196abc.png)

I just want to check I'm not going mad.... this is a standard twill feature or it's something that someone working on this side has added ?  I can't find any reference to it anywhere in the source except the /public/vendor.js file.

**Comment by ifox** [comment]
Hi @adam-jones-net it is a standard field of the medias form field. We are changing it to not be enabled by default in Twill 3 as it is a bit niche. The intention originally was that when using an embedded video player we always want a still image to display before booting the player after user interaction. Of course if your frontend doesn't do anything with it it won't have any effect for your client.

**Reply by adam-jones-net** [reply]
Thanks for confirming @ifox
